www/docs/en/dev/platform_plugin_versioning_ref/index.md

```diff
@@ -22,225 +22,276 @@ toc_title: Manage versions and platforms
 description: How to manage platforms and Cordova CLI versions.
 ---
 
-# Platforms and Plugins Version Management
+## Platforms and Plugins Version Management
+
 From version 4.3.0 onwards, Cordova provides the ability to save and restore platforms and plugins.
 
 This feature allows developers to save and restore their app to a known state without having to check in all of the platform and plugin source code.
 
 When adding a platform or plugin, details about the app's platform and plugin versions are automatically saved in config.xml and package.json. It is possible to add a platform or plugin by editing package.json or config.xml directly, assuming you know the right tags + syntax. It is not possible to remove plugins or platforms in this manner. The recommended method of adding and removing plugins and platforms is with the command line cordova commands `cordova plugin add|remove ...` and `cordova platform add|remove ...` to avoid any out of sync issues.
 
-The 'restore' step happens automatically when a **'cordova prepare'** is issued, making use of information previously saved in the config.xml and package.json files.
+The 'restore' step happens automatically when a **`cordova prepare`** is issued, making use of information previously saved in the config.xml and package.json files.
 
 One scenario where save/restore capabilities come in handy is in large teams that work on an app, with each team member focusing on a platform or plugin. This feature makes it easier to share the project and reduce the amount of redundant code that is checked in the repository.
 
-
 ## Platform Versioning
 
-### Saving platforms
-To save a platform, you issue the following command :
+### Saving Platforms
+
+To save a platform, issue the following command:
 
 ```bash
-$ cordova platform add <platform[@<version>] | directory | git_url>
+cordova platform add <platform[@<version>] | directory | git_url>
 ```
 
-After running the above command, the resulting config.xml looks like :
-
-```xml
-<?xml version='1.0' encoding='utf-8'?>
-    ...
-    <engine name="android" spec="~4.0.0" />
-    ...
-</xml>
+After running the above command, the **`package.json`** should contain something as seen below:
+
+```json
+"cordova": {
+  "platforms": [
+    "android"
+  ]
+},
+"dependencies": {
```

Comment:
Is it `dependencies` or `devDependencies`? It really *should* be `devDependencies` but I suspect not...

Reply:
When looking at a project that I created awhile back with Cordova 9, it put the platform as a `dependencies` and plugins as `devDependencies`.

Reply:
Yay consistency :roll_eyes: